src/uxBlock/select.go

```diff
@@ -84,19 +86,30 @@ type selectModel struct {
 	selected  map[int]struct{}
 	quiting   bool
 	canceled  bool
+
+	width  int
```

Comment:
Won't it brak something starting with value 0? Or the window size event is sent at beggining? If not it should be taken from `uxBlocks.terminalWidth` upon model inittialization, likewise for height.

Reply:
I think it would be more clear to have them not to 0, also tables from `uxBlocks` are not buble tea components, but lipgloss, what you are talking about is `select` interactive component. Altough this MR is releated only to `select`, it would be nice to have it set to right value upon initialization nonetheless.

Reply:
Updated.